AQ3DC/AQ3DC.py

```diff
@@ -351,13 +351,13 @@ def manageDisplayImportExportMeasurementPage(self):
     
     Call by CbImportExportMeasure
     """
-        dic = {
+        dict = {
```

Comment:
It is better to avoid using `dict` as a variable name, since doing so shadows the built-in `dict` class, making it inaccessible. I'd recommend finding alternative names for these temporary variables.

For example this one could be called `choices` or `indexes` or something similar.